docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -45,17 +45,22 @@ If you are using Docker, use an environment variable to enable public dashboards
 - Click on the sharing icon to the right of the dashboard title.
 - Click on the Public Dashboard tab.
 - Acknowledge the implications of making the dashboard public by checking all the checkboxes.
-- Turn on the Enabled toggle.
-- Click `Save Sharing Configuration` to make the dashboard public and make your link live.
+- Click `Generate public URL` to make the dashboard public and make your link live.
 - Copy the public dashboard link if you'd like to share it. You can always come back later for it.
 
+#### Pause access
+
+- Click on the sharing icon to the right of the dashboard title.
+- Click on the Public Dashboard tab.
+- Turn on the `Pause sharing dashboard` toggle.
+- Anyone with the link will not be able to access the dashboard publicly until you make it shareable again.
```

Comment:
Removing the bullet because this is an outcome rather than a step.

```suggestion

The dashboard is no longer accessible, even with the link, until you make it shareable again.
```